timeusage/templates/time_usage.html

```diff
@@ -0,0 +1,21 @@
+{% extends "manage/base.html" %}
+{% load static %}
+{% load dictiterators %}
+{% load i18n %}
+
+{% block head %}
+{{ block.super }}
+	<script src="{% static "filterbar.js" %}"></script>{# extra js for feedback filter tool #}
```

Comment:
Where is `filterbar.js` since I don't see it in this pull request?

Reply:
Actually, `filterbar.js` is a part of the existing mooc-jutut code: 
https://github.com/Aalto-LeTech/mooc-jutut/blob/911ac911c3b8c8e93d9f1ce511f38a1a7c61015b/feedback/static/filterbar.js